packages/core/types/src/http/cart/store/entities.ts

```diff
@@ -80,3 +85,37 @@ export interface StoreCartShippingMethod extends BaseCartShippingMethod {
     shipping_method: StoreCartShippingMethod
   })[]
 }
+export interface StoreCartPromotion {
+  /**
+   * The promotion's ID.
+   */
+  id: string
+  /**
+   * The promotion's code.
+   */
+  code?: string
+  /**
+   * Whether the promotion is applied automatically (without the customer needing to enter a code).
+   */
+  is_automatic?: boolean
```

Comment:
question: Don't think these can be undefined

Reply:
They're defined the same way in `StorePromotion`. Should I fix it in both places?